config/__init__.py

```diff
@@ -200,14 +219,20 @@ def update_status_config(milestones_content):
     'type-webvr',
 ]
 
+# Get AB experiment variation values from the environement.
+# By default, it will serve v1 100% of the time.
+FORM_V1_VARIATION = os.environ.get('FORM_V1_VARIATION') or '0 100'
+FORM_V2_VARIATION = os.environ.get('FORM_V2_VARIATION') or '100 100'
+EXP_MAX_AGE = os.environ.get('EXP_MAX_AGE', None)
+
 # AB testing config
 AB_EXPERIMENTS = {
     'exp': {
         'variations': {
-            'form-v1': (0, 100),
-            'form-v2': (100, 100)
+            'form-v1': get_variation_from_env(FORM_V1_VARIATION),
```

Comment:
Looking again, I don't think this will break because we provide some default values `FORM_V1_VARIATION = os.environ.get('FORM_V1_VARIATION') or '0 100'` -- this should serve v1 100% of the time. We just need to add some logging here if it raises.

Reply:
It will break take this scenario:

1. environment variable defined as `export FORM_V1_VARIATION='0 a'`
2. `FORM_V1_VARIATION = os.environ.get('FORM_V1_VARIATION', '0 100')` is valid and `FORM_V1_VARIATION` will be `0 a`
3. The code goes to `AB_EXPERIMENTS` which contains `'form-v1': get_variation_from_env(FORM_V1_VARIATION)`
4. This will raise an exception. There is no default value at this stage.

The default value applies only if environment variable doesn't exist.

Reply:
Ah. Right.